cwms_radar_api/src/main/java/cwms/radar/api/TimeSeriesController.java

```diff
@@ -88,10 +107,16 @@ public void getAll(Context ctx) {
             String begin = ctx.queryParam("begin");
             String end = ctx.queryParam("end");
             String timezone = ctx.queryParam("timezone");
+            // The following parameters are only used for jsonv2
+            String cursor = ctx.queryParam("cursor",String.class,ctx.queryParam("page",String.class,"").getValue()).getValue();
+            int pageSize = ctx.queryParam("pageSize",Integer.class,ctx.queryParam("pagesize",String.class,Integer.toString(defaultPageSize)).getValue()).getValue();
 
+            String acceptHeader = ctx.header("Accept");
+            ContentType contentType = Formats.parseHeaderAndQueryParm(acceptHeader, null);
 
             switch(format){
                 case "json": {ctx.contentType(Formats.JSON); break;}
+                case "jsonv2": {ctx.contentType(Formats.JSONV2); break;}
```

Comment:
This isn't necessary here, and new formats shouldn't be handled in the query param. See the OfficeController for what we're moving to.

Reply:
Fair enough. I did add the accept-format method. I added the above so it's easy to see in a browser, but I can remove it.